remotedev/create.py

```diff
@@ -0,0 +1,180 @@
+# This script takes one argument: the name of the GitHub user for whom you want
+# to create a Zulip developer environment. Requires Python 3.
```

Comment:
Can you add a comment like "This script creates a droplet on digitalocean for interns and others to do remote development."  Either that or create a small "readme.md" in this directory?